charts/skypilot/values.yaml

```diff
@@ -81,12 +81,20 @@ ingress:
   authSecret: null
   # Basic auth credentials in format "username:encrypted_password" (only used if ingress.authSecret is not set)
   authCredentials: "username:$apr1$encrypted_password"
+  # The base path of API server, you may use differenet path to expose multiple API server through a unified ingress-controller
+  path: '/'
+  # GKE load balancer used for ingress does not work with websocket connections
+  # (required for our k8s SSH tunneling), in such case NodePort should be enabled
+  # to exposes the ingress controller.
+  # Note: nodePort svc will only be created when both ingress.nodePortEnabled=true AND ingress-nginx.enabled=true
+  nodePortEnabled: true
```

Comment:
keep the schema instead of using `nodePort.httpNodePort` and `nodePort.enabled` in favor of backward compatibility